.config/tsaoptions.json

```
@@ -0,0 +1,10 @@
+{  
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\SDL",
```

```suggestion
    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition\\Source-build",
```

I don't believe this path exists and will need to be created.  If you don't have permission to create it Nikola can help.